src/main/com/smithsmodding/armory/client/render/tileentity/TileEntityRendererForge.java

```diff
@@ -20,7 +19,8 @@
 public class TileEntityRendererForge extends TileEntitySpecialRenderer<TileEntityForge> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)